test/check-application

```diff
@@ -1329,6 +1329,9 @@ class TestApplication(testlib.MachineCase):
         self.performContainerAction("swamped-crate", "Start")
         b.wait(lambda: self.getContainerAttr("swamped-crate", "State") in 'Running')
 
+        self.toggleExpandedContainer("swamped-crate")
+        b.wait_not_present(f'#containers-containers tr:contains("{IMG_BUSYBOX}") dt:contains("Latest checkpoint")')
```

Comment:
That's a race condition. After toggling, you first need a positive check for something else in the expanded area, as immediately after clicking the toggle icon the "not present" check will always be true for a while.

Reply:
Fixed